test/integration/tests/utilities.js

```diff
@@ -181,6 +182,11 @@
       .then(() => syncStore.clear());
   }
 
+  function cleanAndPopulateCollection(collectionName, entities) {
```

Comment:
@tsvetomir-nedyalkov I noticed `cleanUpCollectionData()` and `saveEntities()` are almost always used together, so I added this helper method.